### ISSUE TITLE
Output of the material list

### ISSUE THREAD

**Issue by DanSman** [dragotin/kraft#125]
Hey

We used to calculate the list of materials separately. Now I'm setting up just one time template which includes the materials per sqm as well. 

The problem is, I don't know what is included and what is not. I wrote myself a list in the text box of the template, but it would be better to get an output of the list that might include how many liters, pieces ... I need.

**Comment by dragotin** [comment]
Yes, the idea was indeed to use the material calculation data of the item templates to calculate a material list for the entire document. Nobody indicated the wish to really use that so far, so it was not implemented yet.

What is there: The item templates can have a list of required material per 1 unit in the calculation data.
What needs to be done: When an item is placed into a document from an item template, the material list is not yet stored with the doc. And later with the archived doc.

I'd love to do that at some point, but tbh that is still far off.

**Comment by Dan-Sun** [comment]
I wish I would be able to help you with that. It would be a very simple quote :) But it's also fine to do it manually. It's just a pity that the option is not really usable.

Anyway, happy new year ;)

**Comment by DanSman** [comment]
I wish I could help you with that. It would be a very simple quote :) But it's also fine to do it manually. It's just a pity that the option is not really usable.

Anyway, happy new year ;)

**Comment by DanSman** [comment]
I wish I would be able to help you. It would be a very simple quote :) But it's also fine to do it manually. It's just a pity that the option is not really usable.

Anyway, happy new year ;)

**Comment by DanSman** [comment]
Would be a donation change `still far off` to `done`? Maybe we can start some kind of fundraiser? Maybe you can give me a hint where to start and I will try my best ;)
